src/Actions/CopyAssetUrl.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Statamic\Actions;
+
+use Statamic\Contracts\Assets\Asset;
+
+class CopyAssetUrl extends Action
+{
+    protected $confirm = false;
+
+    public static function title()
+    {
+        return __('Copy URL');
+    }
+
+    public function visibleTo($item)
+    {
+        return $item instanceof Asset;
+    }
+
+    public function visibleToBulk($items)
+    {
+        return false;
+    }
+
+    public function authorize($authed, $user)
+    {
+        return $authed->can('sendPasswordReset', $user);
```

Comment:
what does this have to do with the password reset?

Reply:
Hah, good point. Copied from my other PR, will update it tomorrow.